drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -20,6 +20,25 @@ config NRF_WIFI_IF_AUTO_START
 	bool
 	default y
 
+config NRF_WIFI_PATCHES_EXT_FLASH
+	bool "Store nRF700x FW patches in external flash"
+	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
+	# For accessing external flash
+	select FLASH
+	# QSPI_NOR is enabled based on DTS
+	select SPI_NOR if BOARD_NRF7002DK_NRF5340_CPUAPP
+	# For relocation code to external flash
+	select XIP
+	select BUILD_NO_GAP_FILL
+	select CODE_DATA_RELOCATION
+	select HAVE_CUSTOM_LINKER_SCRIPT
+
+if NRF_WIFI_PATCHES_EXT_FLASH
+config CUSTOM_LINKER_SCRIPT
+	string "Custom linker script for nRF700x FW patches"
```

Comment:
Better to omit this line in an override to get an error when the original Kconfig name changes.